apps/core/scenes/results/Results.js

```diff
@@ -6,6 +6,7 @@ import { QueryRenderer, graphql } from '@kiwicom/margarita-relay';
 import { StyleSheet } from '@kiwicom/universal-components';
 import * as DateFNS from 'date-fns';
 import { SearchParamsSummary } from '@kiwicom/margarita-components';
+import { mobileTokens } from '@kiwicom/margarita-utils';
```

Comment:
I definitely don't insist on it and it's just a quick idea, but what about moving it to separate package? Something like:
`import { mobileTokens } from '@kiwicom/margarita-design-tokens';`

Reply:
Why not, but let's wait for notes from the others.